docker/python3-scientific/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Ambrose Carr <mail@ambrosejcarr.com>
 LABEL software="python 3.6.2"
 LABEL description="python 3.6.2 with pysam and scsequtil"
 
-RUN pip3 install scsequtil==0.0.5 \
+RUN pip3 install scsequtil==0.0.6 \
```

Comment:
You might want to define versions for the subsequent packages too. Changes in versions of numpy or scipy can create lots of problems.